packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -2177,6 +2188,8 @@ class WebpackCLI implements IWebpackCLI {
   ): Promise<WebpackCompiler> {
     if (typeof options.defineProcessEnvNodeEnv === "string") {
       process.env.NODE_ENV = options.defineProcessEnvNodeEnv;
```

Comment:
Ideally we need to remove this line, because `defineProcessEnvNodeEnv` should set `NODE_ENV` only for runtime, but I am afraid some people can migrate on `define-process-env-node-env` and we can do breaking change for them, so can you add `TODO` here and we will solve it in the next breaking change, thank you

Reply:
Yeah definitely. I didn't want to make a breaking change on the thing that was literally just added! I'll add a TODO :)